vaadin-number-field.html

```diff
@@ -0,0 +1,199 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../vaadin-themable-mixin/vaadin-themable-mixin.html">
+<link rel="import" href="vaadin-text-field.html">
+
+<dom-module id="vaadin-number-field-default-theme">
+</dom-module>
+
+<dom-module id="vaadin-number-field-template">
+  <template>
+    <style>
+      :host {
+        width: 120px;
+      }
+
+      :host([has-controls]) [part="value"] {
+        text-align: center;
+      }
+
+      /* Value control styles */
+      [part="decrease-button"],
+      [part="increase-button"] {
+        display: inline-block;
+        cursor: pointer;
+        padding: 0 8px;
+        opacity: 0.5;
+      }
+
+      [part="decrease-button"][hidden],
+      [part="increase-button"][hidden] {
+        display: none;
+      }
+
+      [part="decrease-button"]::before {
+        content: "–"; /* en dash */
```

Comment:
Use the proper minus symbol instead: −

For more info, if you’re a typogeek :)
http://www.punctuationmatters.com/the-difference-between-a-dash-and-a-minus-sign/

Reply:
Done, thanks!